src/backends/gerrit.js

```diff
@@ -12,17 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import chromeAsync from '../chromeasync'
+
 export class GerritBackend {
   constructor(site) {
     this.site_ = site;
   }
 
   fetch() {
-    return this.ensureLogin_().then(this.fetchAll_.bind(this));
+    return this.ensureLogin_()
+      .then((selfAddress) => this.fetchAll_(selfAddress));
   }
 
   fetchAll_(selfAddress) {
-    var queries = [
+    let queries = [
```

Comment:
Thanks for catching, I was not aware I still used var here. But could you use const if it's possible?